src/swarmus_ros_simulation/launch/slam_simulation.launch

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" ?>
+<launch>
+    <!-- Arguments-->
+    <arg name="model" default="$(find swarmus_ros_description)/urdf/pioneer.urdf.xacro"/>
+    <arg name="robot_0_name" default="pioneer_0"/>
```

Comment:
I think we should used `robot_name` instead of `robot_0_name` because this code is not specific to robot 0